project/AkkaDisciplinePlugin.scala

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka
+
+import sbt._
+import Keys.{scalacOptions, _}
+import sbt.plugins.JvmPlugin
+
+/**
+  * Initial tests found:
+  * `akka-actor` 151 errors with `-Xfatal-warnings`, 6 without the flag
+  */
+object AkkaDisciplinePlugin extends AutoPlugin with ScalafixSupport {
+ 
+  import scalafix.sbt.ScalafixPlugin
+
+  override def trigger: PluginTrigger = allRequirements
+  override def requires: Plugins = JvmPlugin && ScalafixPlugin
+
+  override lazy val projectSettings = disciplineSettings
+
+  lazy val scalaFixSettings = {
+    import scalafix.sbt.ScalafixPlugin.autoImport._
+    Seq(scalacOptions in Compile ++= Seq("-Yrangepos"))
```

Comment:
I would be for using the new slash syntax which has been in sbt for quite some time now. With that, this would become:

```scala
Compile / scalacOptions
```